docs/UserGuide.md

```diff
@@ -72,295 +82,468 @@ Information that is important to know to prevent unexpected or undefined behavio
+|------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------|
+| `a/ADDRESS`                  | Any characters                                                                                                                                               | 311, Clementi Ave 2, #02-25 |
+| `c/COMPANY_INDEX`            | A positive integer (e.g. 0, 1, 2…) that is smaller than the size of the list of companies currently displayed<br/><br>Takes in a maximum value of 2147483647 | 1, 10, 26                   |
+| `d/DESCRIPTION`              | Any characters                                                                                                                                               | A streaming service company |
+| `e/EMAIL `                   | local-part@domain                                                                                                                                            | johnd@example.com           |
+| `i/INTERNSHIP_INDEX`         | A positive integer (e.g. 0, 1, 2…) that is smaller than the size of the list of companies currently displayed<br/><br>Takes in a maximum value of 2147483647 | 1, 10, 26                   |
+| `n/COMPANY_NAME`             | Alphanumeric with spaces and the punctuation & , . -                                                                                                         | Apple Inc., AT&T            |
+| `n/PERSON_NAME `             | Alphabetical with spaces                                                                                                                                     | John Doe                    |
+| `n/KEYWORD `                 | Alphanumeric without spaces and the punctuation & , . -                                                                                                      | John, Doe                   |
+| `p/PHONE_NUMBER`             | Numerical                                                                                                                                                    | 91234567                    |
+| `s/SCHEDULED_INTERVIEW_TIME` | `DD-MM-YYYY HH:mm`                                                                                                                                           | 20-02-2024 09:45            |
+| `t/TAG`                      | Alphanumeric                                                                                                                                                 | CS2103T                     |
+| `start/START_TIME`           | `DD-MM-YYYY HH:mm`                                                                                                                                           | 20-02-2024 09:45            |
```

Comment:
I think just to be safe we can put like, "where DD is day, MM is month, YYYY is year..." just in case cs2101 has a problem with it

Reply:
sure